build/tzdbupdate/update-2.4.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -e
```

Comment:
nit: if it works everywhere, you probably want `set -u` as well, to fail on accesses to unknown variables.

(it's also fairly common to just write that as `set -eu -o pipefail`, in one line.)